File: URI/1 - INICIANTE/C/1065 - ParesEntreCincoNumeros.c
#include <stdio.h>

int main()
{
	int a, b, c, d, e, f, pares;
	
	scanf(" %d", &a);
	scanf(" %d", &b);
	scanf(" %d", &c);
	scanf(" %d", &d);
	scanf(" %d", &e);
	
	if (a % 2 == 0){pares +=1;}
	if (b % 2 == 0){pares +=1;}
	if (c % 2 == 0){pares +=1;}
	if (d % 2 == 0){pares +=1;}
	if (e % 2 == 0){pares +=1;}
	
	printf("%d valores pares\n", pares);	
	return 0;
}

File: URI/1 - INICIANTE/Python/1065 - ParesEntreCincoNumeros.py
a = int(input())
b = int(input())
c = int(input())
d = int(input())
e = int(input())
pares = 0
if a % 2 == 0:
    pares += 1
if b % 2 == 0:
    pares +=1
if c % 2 == 0:
    pares += 1
if d % 2 == 0:
    pares += 1
if e % 2 == 0:
    pares += 1


print('{} valores pares'.format(pares))
